jdi-light-examples/src/main/java/io/github/epam/EpamGithubStaticSite.java

```diff
@@ -15,7 +15,7 @@
 import java.util.List;
 
 @JSite("https://jdi-testing.github.io/jdi-light/")
-public class EpamGithubSite {
+public class EpamGithubStaticSite {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'EpamGithubStaticSite' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/app/jdi-testing/jdi-light/pullRequest?prid=4482156)

Reply:
class name EpamGithubStaticSite does match such mask